### PR TITLE
To dolistmarkup

### PR DESCRIPTION
**Pull request by Johnr302**
please check added content and formatting

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "terminal.integrated.fontSize": 16
+}
```
Comment (d48) [inline .vscode/settings.json:3]: add this file to `.gitignore`. You don't want this in your code repo if it's only your local configuration settings

File: to-do.html [hunk 2]
```diff
@@ -1,40 +1,42 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <title> Pjohn's to do list!</title>
-</head>
-<body>
+<html lang="en">
```
Comment (d48) [inline to-do.html:2]: great

File: to-do.html [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>Pjohn's to do list help!</title>
+    <link rel="stylesheet" href="styles.css" />
```
Comment (d48) [inline to-do.html]: make sure this links to the correct stylesheet file

File: to-do.html [hunk 4]
```diff
@@ -1,40 +1,42 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <title> Pjohn's to do list!</title>
-</head>
-<body>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Pjohn's to do list help!</title>
```
Comment (d48) [inline to-do.html]: why does it say `help!`

File: to-do.html [hunk 5]
```diff
@@ -1,40 +1,42 @@
...
-            <li> <input type="text" id="l14"></li>
-            <li> <input type="text" id="l15"></li>
-            <li> <input type="text" id="l16"></li>
-            <li> <input type="text" id="l17"></li>
-            <li> <input type="text" id="l18"></li>
-            <li> <input type="text" id="l19"></li>
-            <li> <input type="text" id="l20"></li>
-        </ul>
+      <ul>
+        <li><input type="text" id="l1" /></li>
```
Comment (d48) [inline to-do.html:18]: tell me more about this ending `/`

File: to-do.html [hunk 6]
```diff
@@ -1,40 +1,42 @@
...
+        <li><input type="text" id="l18" /></li>
+        <li><input type="text" id="l19" /></li>
+        <li><input type="text" id="l20" /></li>
+      </ul>
     </div>
-    <footer> information here </footer>
-</body>
-</html>
\ No newline at end of file
+    <footer>information here</footer>
```
Comment (d48) [inline to-do.html]: update this with the actual information

**Review by d48 — CHANGES_REQUESTED** [review]

File: to-do.html (hunk 5 above)
Comment (Johnr302) [reply to-do.html:18]: it closes the input tag inside the li tag. the code formatter puts them in there. web examples do not show the /

Comment (Johnr302) [reply to-do.html:18]: rules in XML required all tags to be opened and closed

File: to-do.css [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  width: 215px;
+}
+button {
+  box-align: left;
+  font-size: 15px;
+  margin: 10px auto;
+}
+html {
+  padding-top: 25px;
+  /*background-image: url(images/bg_page.png);*/
```
Comment (d48) [inline to-do.css]: didn't notice this commented out code. Remove if you're not going to use it

**Comment by d48** [comment]
I just have final 2 comments. If you update those I'll merge this pull request 👍

**Review by d48 — APPROVED** [review]
all looks bueno 👍

**Pull request merged**